src/components/expression/expression-renderer.js

```diff
@@ -149,24 +120,46 @@ genotet.utils.inherit(genotet.ExpressionRenderer, genotet.ViewRenderer);
 genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_HEIGHT = 150;
 
 /** @const {number} */
+genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_LEGEND_HEIGHT = 25;
+
+/** @const {number} */
+genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_LEGEND_MARGIN = 10;
+
+/** @const {number} */
 genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_MARGIN = 40;
 
-/*
- // gene profile properties
- this.legendHeight = 20;
- this.legendMarginDown = 5;
- this.legendMarginLeft = 10;
- this.legendMarginTop = 10;
- this.legendBoxSize = 10;
- this.legendFontSize = 7;
- this.legendBoxTextGap = 5;
-
- // heatmap properties
- this.heatmapMargin = 3;
- this.heatmapFontSize = 8.5;
- this.heatmapFontHSize = 8.5;
- */
+/** @const {number} */
+genotet.ExpressionRenderer.prototype.DEFAULT_HEATMAP_GRADIENT_HEIGHT = 30;
+
+/** @const {number} */
+genotet.ExpressionRenderer.prototype.GENE_LABEL_WIDTH_FACTOR = 8.725;
 
+/** @const {number} */
+genotet.ExpressionRenderer.prototype.CONDITION_LABEL_HEIGHT_FACTOR = 6.501175;
```

Comment:
I calculated it in some magic way that I cannot remember now... = = Okay, let's leave it temporarily.